Kickstarter-iOS/ViewModels/RootViewModel.swift

```diff
@@ -477,19 +489,14 @@ extension TabBarItem: Equatable {
 }
 
 private func activitiesBadgeValue(with value: Int?) -> String? {
-  let maxBadgeValue = 99
+  let isVoiceOverRunning = AppEnvironment.current.isVoiceOverRunning()
   let badgeValue = value ?? 0
+  let maxBadgeValue = !isVoiceOverRunning ? 99 : badgeValue
   let clampedBadgeValue = min(badgeValue, maxBadgeValue)
 
   guard clampedBadgeValue > 0 else { return nil }
 
-  guard badgeValue > maxBadgeValue else {
-    return "\(clampedBadgeValue)"
-  }
-
-  return localizedString(
-    key: "activities_badge_value_plus",
-    defaultValue: "%{activities_badge_value}+",
-    substitutions: ["activities_badge_value": "\(clampedBadgeValue)"]
-  )
+  return (badgeValue > maxBadgeValue) && !isVoiceOverRunning
```

Comment:
Is `badgeValue > maxBadgeValue` ever going to happen?

Reply:
Yes, the push notification can be any amount, this is the check to see if it's say `100` or more in the push payload, then display `99+`. When voiceover is enabled, we show the actual value from the payload without any `+`.